src/code42cli/cmds/profile.py

```diff
@@ -103,7 +113,9 @@ def create(name, server, username, password, disable_ssl_errors):
 @username_option()
 @password_option
 @disable_ssl_option
-def update(name, server, username, password, disable_ssl_errors):
+@debug_option
+@totp_option
```

Comment:
Yeah, if a user doesn't have a password saved and they don't provide the `--password` option, it'll prompt them if they want to save. Which won't use the `--totp` if they happen to have passed it, it'll just force a prompt (as it'll fail first but detect LOCAL_2FA LoginConfig and then prompt for totp). 

I'd rather ignore the option input since it'll prompt for it anyway rather than raise an error and tell the user they can't provide it as an option.